SpringProject/src/it/java/com/gradproject2019/user/UserControllerIT.java

```diff
@@ -38,13 +37,18 @@
     private String baseUri;
     private URI uri;
     private URI uri2;
+    private URI uri3;
+    private URI uri4;
 
     @Before
     public void setUp() throws URISyntaxException {
         universalSetUp();
         baseUri = "http://localhost:" + testServerPort + "/users";
         uri = new URI(baseUri);
         uri2 = new URI(baseUri + "/" + savedUser.getId());
+        uri3 = new URI(baseUri  +"?manager_id=" + savedManager.getId());
+        uri4 = new URI(baseUri + "?manager_id=" + savedUser.getId());
```

Comment:
1. Do you need this and `uri3`?
2. uri1, uri2, uri3 etc. are not very clean variable names as it doesn't tell you anything about the URI itself - can you change these to be more descriptive?

Reply:
Yep :)